src/components/agencyspecific.ts

```diff
@@ -0,0 +1,148 @@
+}
+
+export function fixStationName(name: string) {
+    console.log(name)
+    let fixPatterns = [
+        ['L.A. Union Station', 'Los Angeles'],
+        ['North Hollywood Station', 'NoHo'],
+        ['North Hollywood Station G - Line', 'NoHo'],
+        ['Chatsworth Station G - Line', 'Chatsworth'],
+        ['Union Station', 'UnionSta'],
+        ['Wilshire / Western Station', 'Wil/Wstrn'],
+        ['Wilshire / Vermont Station', 'Wilshire Ctr'],
+        ['Downtown Long Beach Station', 'Long Bch'],
+        ['APU / Citrus College Station', 'Azusa'],
+        ['Redondo Beach Station', 'Redondo Bch'],
+        ['Norwalk Station', 'Norwalk'],
+        ['Downtown Santa Monica Station', 'S Monica'],
+        ['Atlantic Station', 'East LA'],
+        ['Expo / Crenshaw Station', 'Expo/Crnshw'],
+        ['Westchester / Veterans Station', 'Westchester'],
+        ['University & College', 'City Heights'],
+        ['12th & Imperial', '12th/Imp\'l'],
+        ['El Cajon / Arnele', 'El Cajon'],
+        ['Crenshaw C-Line Station', 'Crenshaw'],
+        ['Otay Mesa Transit Center', 'Otay Mesa'],
+        ['Redmond Technology', 'Redmond Tech'],
+        ['South Bellevue', 'S Bellevue'],
+        ['Downtown SD', 'Downtown'],
+        ['Ucsd', 'UCSD'],
+        ['Sdsu', 'SDSU'],
+        ['Utc', 'UTC'],
+        ['Va / Ucsd', 'UTC'],
+        ['UTC/VA Med Ctr', 'UTC'],
+        ['Old Town to Airport Shuttle', 'Airport'],
+    ]
+
+    for (let i = 0; i < fixPatterns.length; i++) {
+        if (name == (fixPatterns[i][0])) {
+            return fixPatterns[i][1]
+        }
+    }
+
+    return fixStationNameLong(name)
+}
+
+export function fixStationNameLong(name: string) {
+    let fixPatterns = [
+        ['Union Station', 'Union Station'],
+        ['APU / Citrus College Station', 'APU/Citrus'],
+        ['Atlantic Station', 'East Los Angeles'],
+        ['Wilshire / Western Station', 'Wilshire/Western'],
+        ['Wilshire / Vermont Station', 'Wilshire/Vermont'],
+        ['12th & Imperial', '12th & Imperial'],
+        ['North Hollywood Station G - Line', 'North Hollywood'],
+        ['Old Town to Airport Shuttle', 'Airport'],
+        ['Sabre Springs & Penasquitos Transit Station', 'Sabre Springs/Peñasquitos'],
+        ['Clairemont Mesa Bl & Complex Dr', 'Kearny Mesa'],
+        ['I-15 Centerline Sta & University Av', 'City Heights'],
+        ['I-15 Centerline Sta & El Cajon Bl', 'Boulevard'],
+        ['A Pacific Beach', 'Pacific Beach - 9A'],
+    ]
+
+    for (let i = 0; i < fixPatterns.length; i++) {
+        if (name == (fixPatterns[i][0])) {
+            return fixPatterns[i][1]
+        }
+    }
+
+    return name.replace(' Transit Station', '').replace(' Station', '').replace(' Metrolink', '').replace('Rapid ', '').replace(' Amtrak', '').replace(' Sta', '').replace('Local', '').replace('Express', '').replace('Downtown ', '').replace('Transportation Center', '').replace('Transit Center', '').replace('Transit Ctr', '').trim()
```

Comment:
Replacing the word Amtrak and Metrolink and Metrolink Station probably make sense, but why are we dropping Downtown?

Reply:
Again, my bad, copied from sign mode